framework/src/arastorage/storage_interface.c

```diff
@@ -84,9 +84,14 @@ struct index_record_s {
 };
 
 /****************************************************************************
-* Public Functions
+* Public Variables
 ****************************************************************************/
 #ifdef CONFIG_ARASTORAGE_ENABLE_WRITE_BUFFER
+struct insert_buffer_s g_storage_write_buffer;
```

Comment:
If this variable `g_storage_write_buffer` uses only in this file, how about adding `static`?